src/jsxstyle-components.js

```diff
@@ -0,0 +1,9 @@
+// Mostly copied from:
+// https://github.com/threepointone/glamor/blob/
+// 5e7d988211330b8e2fca5bb8da78e35051444efd/src/jsxstyle.js
+/*
+ * This is used to generate the helper functions
+ * like: glamorous.block and the <glamorous.Block /> elements
+ */
+
+export default ['view', 'block', 'inlineBlock', 'flex', 'row', 'column']
```

Comment:
> Actually, because we're going to be constructing things ourselves (rather than using glamor's implementation) perhaps this should be more of a config:

I had thought about a solution like this at one point, and I like this approach a little better even.


> I'm also thinking that it may be better to not put them directly on glamorous but instead scope them to a property on glamorous. Maybe glamorous.display? Because they're all display-specific? So you'd do:

The other components like `a` and `div` are right on `glamorous` though. It's a little weird that some components would be scoped lower than others. You'd have to find where we shoved all these elements. 😄 

If we're going to put these on a `glamorous.display` perhaps we should put them all there?